app/services/invoice_rebilling_initialise.service.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/**
+ * @module InvoiceRebillingInitialiseService
+ */
+
+const { InvoiceModel } = require('../models')
+const { RebillingInvoicesPresenter } = require('../presenters')
+
+class InvoiceRebillingInitialiseService {
+  /**
+   * Creates two empty invoices on a given bill run for rebilling purposes: a 'cancel' invoice and a 'rebill' invoice.
+   * The cancel invoice is the invoice which will cancel out the invoice being rebilled, and the rebill invoice is the
+   * one that will rebill the the customer. They can be identifed by rebillType `C` and `R` respectively. Both invoices
+   * will be linked to the original invoice by rebilledInvoiceId, which contains the id of the original invoice.
+   *
+   * @param {module:BillRunModel} billRun An instance of `BillRunModel` of the bill run to add the invoices to.
+   * @param {module:InvoiceModel} invoice An instance of `InvoiceModel` of the invoice to be rebilled.
+   *
+   * @returns {Object} A reponse object containing `cancelInvoice`, `rebillInvoice` and `response`.
+   * @returns {module:InvoiceModel} `resonse.cancelInvoice` The cancel invoice.
+   * @returns {module:InvoiceModel} `response.rebillInvoice` The rebill invoice.
+   * @returns {Array} `response.response` An array containing the cancel invoice id and type `C`, and the rebill invoice
```

Comment:
I'm not sure this description is correct based on what the presenter generates. Isn't it another `Object` that looks like this?

```javascript
{
  invoices: [
    { id: '0ca40d7f-552e-404a-b931-a2dd7b0e1ff4', rebilledType: 'C' },
    { id: 'b9680a58-ff31-4a14-9dda-eb696f7c5159', rebilledType: 'R' }
  ]
}
```

Reply:
Good spot, yes it is! 🤦